src/Microsoft.Health.Dicom.Core/Features/Query/Model/FilterConditions/CustomTagFilterDetails.cs

```diff
@@ -0,0 +1,66 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Dicom;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+
+namespace Microsoft.Health.Dicom.Core.Features.Query
+{
+    /// <summary>
+    /// Details required to create a custom tag filter condition
+    /// </summary>
+    public class CustomTagFilterDetails : IEquatable<CustomTagFilterDetails>
```

Comment:
Can we reuse CustomTagStoreEntry here? I see the only new property here is DicomTag which can be derived

Reply:
VR and Status are also not used!

---
In reply to: [582547138](https://github.com/microsoft/dicom-server/pull/567#discussion_r582547138) [](ancestors = 582547138)